pcdsdevices/sqr1.py

```diff
@@ -0,0 +1,141 @@
+        self.z._sync_setpoints = self.sync_setpoints
+        self.rx._sync_setpoints = self.sync_setpoints
+        self.ry._sync_setpoints = self.sync_setpoints
+        self.rz._sync_setpoints = self.sync_setpoints
+
+    def sync_setpoints(self):
+        print("sync_setpoint")
+        self.x.setpoint.put(self.x.readback.get())
+        self.y.setpoint.put(self.y.readback.get())
+        self.z.setpoint.put(self.z.readback.get())
+        self.rx.setpoint.put(self.rx.readback.get())
+        self.ry.setpoint.put(self.ry.readback.get())
+        self.rz.setpoint.put(self.rz.readback.get())
+
+    def multi_axis_move(self,
+                        x_sp: float = None,
+                        y_sp: float = None,
+                        z_sp: float = None,
+                        rx_sp: float = None,
+                        ry_sp: float = None,
+                        rz_sp: float = None,
+                        wait: bool = True,
+                        timeout: float = 10.0
+                        ):
+        x_sp = self.x.readback.get() if x_sp is None else x_sp
+        y_sp = self.y.readback.get() if y_sp is None else y_sp
+        z_sp = self.z.readback.get() if z_sp is None else z_sp
+        rx_sp = self.rx.readback.get() if rx_sp is None else rx_sp
+        ry_sp = self.ry.readback.get() if ry_sp is None else ry_sp
+        rz_sp = self.rz.readback.get() if rz_sp is None else rz_sp
+
+        x_status = self.x.move(x_sp,
+                               wait=False,
+                               timeout=timeout,
+                               sync_enable=False)
+        y_status = self.y.move(y_sp,
+                               wait=False,
+                               timeout=timeout,
+                               sync_enable=False)
+        z_status = self.z.move(z_sp,
+                               wait=False,
+                               timeout=timeout,
+                               sync_enable=False
+                               )
+        rx_status = self.rx.move(rx_sp,
+                                 wait=False,
+                                 timeout=timeout,
+                                 sync_enable=False)
+        ry_status = self.ry.move(ry_sp,
+                                 wait=False,
+                                 timeout=timeout,
+                                 sync_enable=False)
+        rz_status = self.rz.move(rz_sp,
+                                 wait=False,
+                                 timeout=timeout,
+                                 sync_enable=False)
+
+        status = AndStatus(
+            x_status,
+            AndStatus(y_status,
+                      AndStatus(z_status,
+                                AndStatus(rx_status,
+                                          AndStatus(ry_status,
+                                                    rz_status)))))
+        return status_wait(status) if wait else status
```

Comment:
This makes it so we sometimes return `None` and sometimes return a status object. Usually it's best to always return the same type when possible.